sdk/src/loom2d/Display/DisplayObject.ls

```diff
@@ -167,7 +167,21 @@ package loom2d.display
          */
         public native function set cacheAsBitmap(value:Boolean);
         public native function get cacheAsBitmap():Boolean;
-        
+
+        /**
+         * If true, cache textures will be resized along with scale, else the cache will remain
+         * the same size even if the scale changes.
+         */
+        public native function set cacheDoApplyScale(value:Boolean);
+        public native function get cacheDoApplyScale():Boolean;
+
+        /**
+          * If true, cache textures will be forced to be sizes of power of two. This can lead to
+          * performance benefits on some GPUs.
+          */
+        public native function set cacheUseTexturesPow2(value:Boolean);
+        public native function get cacheUseTexturesPow2():Boolean;
```

Comment:
Maybe use `Pot` instead of `Pow2` to be consistent with another API in RenderTexture that also references power of two textures `supportsNonPotDimensions`?